src/lib/y2partitioner/dialogs/encrypt_password.rb

```diff
@@ -0,0 +1,26 @@
+require "yast"
+require "cwm/dialog"
+require "y2partitioner/widgets/encrypt_password"
+
+module Y2Partitioner
+  module Dialogs
+    # Formerly MiniWorkflowStepFormatMount
```

Comment:
to be honest I prefer here to have description of class. link to former implementation is fine, but what is purpose is better.